src/style.scss

```diff
@@ -22,6 +22,7 @@ $auro-inset-directions:'--squish';
 
 // focus-visible mixin
 @mixin focus-visible {
+  outline: 3px solid transparent;
```

Comment:
```suggestion
%focus-visible {
  outline: 3px solid transparent;
  box-shadow: inset 0 0 0 1px var(--auro-color-border-focus-on-dark), inset 0 0 0 3px var(--auro-color-background-lightest);

```

Reply:
actually.... I am wrong. There is no need for any of this. 

Kill the silent selector and just put the CSS inside the combination selector below.